chain-signatures/node/src/protocol/sync/mod.rs

```diff
@@ -71,14 +73,17 @@ impl SyncTask {
             mesh_state,
             watcher,
             requests,
+            synced_peer_tx,
         };
         (channel, task)
     }
 
     pub async fn run(mut self) {
         tracing::info!("task has been started");
         let mut watcher_interval = tokio::time::interval(Duration::from_millis(500));
-        let mut broadcast_interval = tokio::time::interval(Duration::from_millis(10000));
+        let mut sync_interval = tokio::time::interval(Duration::from_millis(100));
+        // Broadcast should generally not be necessary.
+        let mut broadcast_interval = tokio::time::interval(Duration::from_secs(3600 * 24));
```

Comment:
We could also remove this entirely, maybe it would keep things simpler. Let me know what you prefer.

Reply:
I think it's fine to keep for now, since it'll clear out inconsistencies in storage